content/pt-br/docs/tasks/debug/debug-application/debug-init-containers.md

```diff
@@ -0,0 +1,115 @@
+---
+title: Contêineres de inicialização de depuração
+content_type: task
+weight: 40
+---
+
+<!-- overview -->
+
+Esta página mostra como investigar problemas relacionados à execução de contêineres de inicialização. 
+As linhas de comando de exemplo abaixo referem-se ao pod como `<pod-name>` e aos contêineres de inicialização como `<init-container-1>` e
+`<init-container-2>`.
+
+## {{% heading "prerequisites" %}}
+
+
+{{< include "task-tutorial-prereqs.md" >}} {{< version-check >}}
+
+* Você deve estar familiarizado com os fundamentos de
+  [contêineres de inicialização](/docs/concepts/workloads/pods/init-containers/).
+* Você deve ter [configurado um contêiner de inicialização](/docs/tasks/configure-pod-container/configure-pod-initialization/#creating-a-pod-that-has-an-init-container/).
+
+<!-- steps -->
+
+## Verificando o status dos contêineres de inicialização
+
+Exiba o status do seu pod:
+
+```shell
+kubectl get pod <pod-name>
+```
+
+Por exemplo, um status de `Init:1/2` indica que uma das duas inicializações de contêineres
+concluíram com sucesso:
+
+```
+NAME         READY     STATUS     RESTARTS   AGE
+<pod-name>   0/1       Init:1/2   0          7s
+```
+
+Consulte [Entendendo sobre o status do pod](#understanding-pod-status) para obter mais exemplos de
+valores de status e seus significados.
+
+## Obtendo detalhes sobre os contêineres de inicialização
+
+Veja informações mais detalhadas sobre a execução da Inicialização de Contêineres:
+
+```shell
+kubectl describe pod <pod-name>
+```
+
+Por exemplo, um pod com duas inicializações de contêineres pode mostrar o seguinte:
```

Comment:
In these sentences:
* "inicialização de contêineres" should be "contêineres de inicialização".
* No need to capitalize those words.